src/app/app.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import {HostsliceComponent} from './hostmod/hostslice/hostslice.component'
+
```

Comment:
i think you needed to rebase. you removed my code

Reply:
we don't need imports on that file . I promise it's still working fine. I see your 6 ___ works! under host name and joined mmddyyyy